docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -384,7 +381,9 @@ If enabled, makes image read-only, with /var/tmp, /tmp and /run a tmpfs (unless
 
 **NOTE:** Podman will automatically copy any content from the image onto the tmpfs
 
-### `RemapGid=`
+### `RemapGid=` (deprecated)
+
+**NOTE:** This key is deprecated and will be removed in a future version of quadlet. Use `UserNS` instead.
```

Comment:
I would just remove the references from the man pages.  We can keep the function around (to prevent breaking users) but just not advertise/extend it.

Reply:
Makes sense, but should I then somehow mark the keys or the handleUserRemap function as deprecated, so that it gets removed at some point in the future?

Reply:
I am personally OK to keep these options around but just not document them.  This way, existing users won't break.

But let's wait for others to chime in.  It's just my 2 cents :)

Reply:
I like the idea of keeping them around but not documenting them

Reply:
Okay, I removed the documentation for the Remap keys.